scipy/stats/morestats.py

```diff
@@ -2692,6 +2693,18 @@ def mood(x, y, axis=0):
         different length along `axis`.
         If `axis` is None, `x` and `y` are flattened and the test is done on
         all values in the flattened arrays.
+    alternative : {'two-sided', 'less', 'greater'}, optional
```

Comment:
From what I see, this is usually in the Notes section (and sometimes we don't describe at all like for `scipy.stats.fisher_exact`). I don't mind seeing it here actually as it does not force the user to go back and forth.

Reply:
Really, it is _most_ common for the alternatives to not be described at all (e.g. [`ttest_ind`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ttest_ind.html#scipy.stats.ttest_ind), [`mannwhitneyu`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.mannwhitneyu.html#scipy.stats.mannwhitneyu)... in SciPy 1.6, I can't think of any tests other than the KS tests that explain what the `alternative`s are). gh-13188 got me thinking about this, and yes, I left the description in the notes there as it had been. But there's not _too_ much precedent I think the norm going forward as we add descriptions of alternatives should be to state it concisely in the Parameters section. If more space or detailed math is needed, it's fine to spill into the Notes. How does that sound?

Reply:
Also I am seeing this from the other PR, a wrong value for this parameter is not tested.

Reply:
Agreed, as I said I actually prefer to have the description here so that the user does not need to go to the notes and then come back here.

Reply:
Great!

Reply:
> Also I am seeing this from the other PR, a wrong value for this parameter is not tested.

For completeness it should be. Oops.